test/aurora.test.ts

```diff
@@ -188,7 +188,7 @@ describe('Aurora', () => {
     });
 
     it('activityStream', () => {
-      const postgresEngineVersion = AuroraPostgresEngineVersion.VER_11_16;
+      const postgresEngineVersion = AuroraPostgresEngineVersion.VER_11_21;
```

Comment:
I assume the version for test doesn't need to match up?

Reply:
This is the minimum supported version.